src/js/controller/controller.js

```diff
@@ -143,8 +144,9 @@ Object.assign(Controller.prototype, {
 
         _model.on('change:scrubbing', function(model, state) {
             if (state) {
+                _stateBeforeScrubbing = _model.get('state');
                 _pause();
-            } else {
+            } else if (_stateBeforeScrubbing === STATE_PLAYING) {
                 _play({ reason: 'interaction' });
```

Comment:
If you scrub while buffering, the player was trying to play. So only checking` _stateBeforeScrubbing === STATE_PLAYING` is incomplete.

```js
_model.on('change:scrubbing', function(model, state) {
  if (state) {
    _resumeAfterScrubbing = _model.get('state') !== STATE_PAUSED;
     _pause();
  } else if (_resumeAfterScrubbing) { 
   _play...
```